doc/build-unix.md

```diff
@@ -96,7 +96,7 @@ Otherwise, if you don't need QR encoding support, use the `-DWITH_QRENCODE=OFF`
 
 Build requirements:
 
-    sudo dnf install gcc-c++ libtool make autoconf automake python3
+    sudo dnf install gcc-c++ cmake python3
```

Comment:
A build tool, such as `make` or `ninja`, is required. So I suggest to keep `make`.

Reply:
done.
(I removed it as I didn't need to install it, as it comes pre-installed on the fedora VM I used to test)